src/components/content/Notifications.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface User {
+    id: string
+    username: string
+  }
+
+interface NotificationProp {
+    author: User,
+    body: string,
+    timestamp: string,
+    readed: boolean
+}
+
+export default function Notifications(entries: NotificationProp[]) {
+    return (
+        <Wrapper>
+            {entries.map((entry, index) => {
+                return (
+                    <Item readed={entry.readed} className={ ( !entry.readed ? "unreaded" : "") }>
```

Comment:
you are already passing the prop to the styled component, consider removing the className and adapt the styling using this prop